src/main/java/org/quiltmc/loader/impl/discovery/ClasspathModCandidateFinder.java

```diff
@@ -116,4 +111,15 @@ public void findCandidates(QuiltLoaderImpl loader, BiConsumer<URL, Boolean> appe
 			}
 		});
 	}
+
+	protected void addModSources(QuiltLoaderImpl loader, Set<URL> modsList, String name) throws IOException {
+		Enumeration<URL> mods = QuiltLauncherBase.getLauncher().getTargetClassLoader().getResources(name);
+		while (mods.hasMoreElements()) {
```

Comment:
I like spaced things. (Put a space between the field declaration and while block)